scene/resources/default_theme/default_theme.cpp

```diff
@@ -114,7 +114,8 @@ static Ref<Texture2D> flip_icon(Ref<Texture2D> p_texture, bool p_flip_y = false,
 	}
 
 	Ref<ImageTexture> texture(memnew(ImageTexture));
-	Ref<Image> img = p_texture->get_data();
+	Ref<Image> orig_img = p_texture->get_data();
+	Ref<Image> img = memnew(Image(orig_img->get_width(), orig_img->get_height(), orig_img->has_mipmaps(), orig_img->get_format(), orig_img->get_data()));
```

Comment:
Can't it be duplicated in a simpler way (like `img->duplicate()`)?

Reply:
I guess it can, yes. Was looking for examples of handling this case in other places, and never checked for the obvious.

Reply:
Fixed.